datahub-web-react/src/app/entity/shared/ExternalUrlButton.tsx

```diff
@@ -54,7 +54,8 @@ export default function ExternalUrlButton({ externalUrl, platformName, entityTyp
                 rel="noreferrer noopener"
                 onClick={sendAnalytics}
             >
-                View in {displayedName} <ArrowRightOutlined style={{ fontSize: 12 }} />
+                {displayedName ? `View in ${displayedName}` : 'View external link'}{' '}
```

Comment:
Curious why we are doing this one? 

View external link doesn't feel very natural.. maybe just me

Reply:
`View source`

Reply:
I'm okay with View source...